src/compose.js

```diff
@@ -0,0 +1,3 @@
+import flowRight from 'lodash.flowright';
```

Comment:
@alitaheri I had the same though and I'm open to removing these. The only library I thought we may one day replace these with is [recompose](https://github.com/acdlite/recompose), which ironically also uses lodash's compose for its implementation. 

Either way I'm going to take it out `compose.js` until we start using it in several places. 

I wanted to talk to you two about `merge.js` though. I'm thinking we should replace all `mergeStyles` with one of the three:

1. merge
2. Object.assign
3. destructuring